src/components/Blocking/SpecificIssue.js

```diff
@@ -99,7 +99,7 @@ const styles = StyleSheet.create({
     paddingRight: 13,
     paddingLeft: 19,
     fontFamily: fonts.inter[400],
-    color: COLORS.white2,
+    color: COLORS.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color has been updated from `COLORS.white2` to `COLORS.white`. As mentioned earlier, ensure that this change doesn't affect the visibility and contrast of the text against its background.
```diff
-    color: COLORS.white2
+    color: COLORS.white
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->